doc/integrator/ngeo.rst.tmpl

```diff
@@ -99,8 +99,10 @@ The sub section is the interface name, and after that we have:
     (but can not be removed via interface configuration).
 * ``constants``: Directly define an ``angular`` constant in the vars file.
 * ``dynamic_constants``: Define an ``angular`` constant from a dynamic values.
-* ``static``: key: Constant name, value: name of the static view that we want to have the URL.
-* ``routes``: key: Constant name, value:
+* ``static``, the ``key`` is the constant name:
+    ``name``: The path of the ressource witch one we want to have the URL.
+    ``append``: Somthing we want to append.
```

Comment:
Typo: Something

Can we say more about what the Something might be?

Reply:
a text ...